### PR TITLE
Add cost in stock exit report

### LINKED ISSUE #4319: Add cost information to the "rapport de sorties de stocks"
We've received this request from our partners in Vanga:

> Dans la farde Rapport, cliquer sur sortie de stock et vous verrez la consommation mensuel complet n'a pas des prix totaux.

There is no price/cost information at all on this report.  Here is what an item looks like:

image
_Fig 1: Stock Exit Report_

We can do better than this.  Let's add:
  1. A column with the unit price.
  2. A column with the total price.
  3. A total field at the bottom of each inventory element.
  4. A grand total at the bottom of the report.

### PR DESCRIPTION
**Pull request by mbayopanda**
This PR fix the issue #4319

Close #4319

### CODE REVIEW

**Comment by kwilu** [comment]
Hello @mbayopanda, it seems like you have referenced #4319 in your pull request description, but you have not referenced them in your commit message description(s). When you reference an issue in a commit message, it automatically closes the corresponding issue when the commit is merged.

Please run `git commit --amend` in your command line client to amend your commit message description with `Fixes #4319.`.

An example of a correctly-formatted commit:
```
commit fabd5e450374c8dde65ec35f02140383940fe146
Author: kwilu
Date:   Sat Mar 18 13:42:40 2017 -0700

    pull requests: Check PR commits reference when issue is referenced.

    Fixes #51.
```

Thank you for your contributions to BHIMA!

**Review by jniles — COMMENTED** [review]
Code looks good.  I need to try it out first before a merge.  I'll do that in a few minutes.

File: server/controllers/stock/reports/stock_exit.report.handlebars [hunk 1]
```diff
@@ -26,14 +26,19 @@
...
+              <th colspan="5"><h3 style="margin: 0px;">{{translate 'STOCK.RECEIPT.EXIT_PATIENTS'}}</h3></th>
+              <th colspan="2" class="text-right">{{currency exitToPatient.cost metadata.enterprise.currency_id}}</th>
+            </tr>
+          </thead>
           {{#each exitToPatient.data as | data |}}
             <thead>
               <tr>
                 <th colspan="4">{{ data.inventory_name }}</th>
                 <th class="text-right">{{ data.inventory_stock_exit_quantity }} <i>({{ data.inventory_unit }})</i></th>
+                <th colspan="2" class="text-right">{{currency data.inventory_stock_exit_cost metadata.enterprise.currency_id}}</th>
```
Comment (jniles) [inline server/controllers/stock/reports/stock_exit.report.handlebars]: This should be `../metadata.enterprise.currency_id` I think.

**Review by jniles — CHANGES_REQUESTED** [review]
@mbayopanda, it looks like the currency doesn't use the enterprise currency properly.  See below:

image

All these numbers should be FC instead of USD as their currency_id.

**Comment by kwilu** [comment]
Hello @mbayopanda, it seems like you have referenced #4319 in your pull request description, but you have not referenced them in your commit message description(s). When you reference an issue in a commit message, it automatically closes the corresponding issue when the commit is merged.

Please run `git commit --amend` in your command line client to amend your commit message description with `Fixes #4319.`.

An example of a correctly-formatted commit:
```
commit fabd5e450374c8dde65ec35f02140383940fe146
Author: kwilu
Date:   Sat Mar 18 13:42:40 2017 -0700

    pull requests: Check PR commits reference when issue is referenced.

    Fixes #51.
```

Thank you for your contributions to BHIMA!

**Review by jniles — APPROVED** [review]
Looks like this is fixed.  Thank you!

image

bors r+

**Pull request merged**